packages/core/src/types.ts

```diff
@@ -1608,6 +1613,10 @@ export interface InterpreterOptions<_TActorBehavior extends AnyActorBehavior> {
 
   sync?: boolean;
 
+  /**
+   * The receptionist key to register this actor under
```

Comment:
This feature has been added as a "system" but we refer here to a "receptionist". The latter term doesn't appear anywhere in the API. Could we stick to a single name for this?